### ISSUE TITLE
[5.2] Plugin "System - Skip-To Navigation": In frontend is the access key description partly illegible

### ISSUE THREAD

**Issue by ghost** [joomla/joomla-cms#45258]
### Steps to reproduce the issue
1. Open in the backend "Home Dashboard > Site > Plugins"
2. Search for "System - Skip-To Navigation"
3. Open plugin, set "Site Section" to "Both", Save & Close
4. Open the frontend
5. Hit the "Tab" key

### Expected result
Access key description looks like in the backend:

<img width="399" alt="Image" src="https://github.com/user-attachments/assets/aceaed12-2198-4dd6-8225-c2d37ad3ead0" />

### Actual result
Looks different in the frontend:

<img width="399" alt="Image" src="https://github.com/user-attachments/assets/9f7fa903-3b6b-4c2c-bc50-1639bd13ab34" />

Example in German:

<img width="450" alt="Image" src="https://github.com/user-attachments/assets/a72a2da3-f610-484f-a1af-6c2fc71207b7" />

### Device and system info

* What version of Joomla were you using?
5.2.5

* What device were you using? (eg, phone, laptop)
MacBook Air, 13-inch

* What web browser were you using? (eg, Chrome, Safari)
Firefox 136.0.4

* If you were using assistive technology, please let us know the device or software you were using (eg VoiceOver, JAWS, sip-and-puff device, etc)
[ no ]

### Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:
[ no ]

**Comment by brianteeman** [comment]
Came to write that its ok for me

![Image](https://github.com/user-attachments/assets/0b704936-3922-4352-92df-741e7953f5de)

But then I realised you are on a Mac. Found the problem in the css. Now just have to work out the correct fix

**Comment by brianteeman** [comment]
Please test #45259
